tests/factory_tests/test_images.py

```diff
@@ -0,0 +1,28 @@
+from io import BufferedReader, BytesIO
+
+import factory
+from django.core.files import File
+
+from incuna_test_utils.factories import images
+
+
+def test_local_file_field():
+    class FileFactory(factory.StubFactory):
+        file = images.LocalFileField()
+
+    file = FileFactory.build().file
+    assert isinstance(file, File)
+    assert isinstance(file.file, BufferedReader)
```

Comment:
`isinstance` can take a `tuple` of types to check against. That might be useful if you can import both types in python 2 and python 3.